src/components/OntologyBrowser/index.tsx

```diff
@@ -44,7 +49,44 @@ const desactivateAllSameTerms = (allSameTerms: string[], transferItems: Transfer
     } else return t;
   });
 
-export const updateSqons = (initialSqon: Sqon, value: string[], selectedField: string): Sqon => {
+const opComponent = (fill: string, operation: string) => {
```

Comment:
maybe change the var name that includes the fact that it returns an icon: eg, "displayOpIcon" or something similar.